apps/server/src/classes/event-loader/EventLoader.ts

```diff
@@ -3,6 +3,7 @@ import { Loaded, OntimeEvent, TitleBlock } from 'ontime-types';
 import { DataProvider } from '../data-provider/DataProvider.js';
 import { getRollTimers } from '../../services/rollUtils.js';
 import { eventStore } from '../../stores/EventStore.js';
+import { editEvent } from '../../services/RundownService.js';
```

Comment:
looks there is an issue with this import.
You might also want to merge master to help with the changes since this file has moved